client/data/fraud-outcomes/test/resolvers.ts

```diff
@@ -0,0 +1,57 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { apiFetch, dispatch } from '@wordpress/data-controls';
```

Comment:
GH Actions thinks it's unused.

Reply:
Removed on [16798c2](https://github.com/Automattic/woocommerce-payments/pull/5841/commits/16798c2014b6b49a82d9d4bdfabd52ec9793072e).